web/scripts/display_policies.js

```diff
@@ -179,8 +179,41 @@ function drawArrow(e){
     connect.setAttribute("to", "#"+targetid)
     connect.setAttribute("tail", "")
     document.body.appendChild(connect)
+    createYaml(sourceid, targetid)
+}
+
+function createYaml(sourceid, targetid){
+    // target is the workload on which the network policy will be applied
+    let source_label = document.getElementById(sourceid).title;
+    let target_label = document.getElementById(targetid).title;
+    target_label = target_label.substring(1, target_label.length);
+    source_label = target_label.substring(0, target_label.length);
+    // Converting to object
+    target_label_split = target_label.split(",");
+    let targetObj = {};
+    for (i=0; i<target_label_split.length; i++) {
+        val = target_label_split[i].split(":")
+        targetObj[val[0]] = val[1]
+    };
+
+    source_label_split = source_label.split(",");
+    let sourceObj = {};
+    for (i=0; i<source_label_split.length; i++) {
+        val = source_label_split[i].split(":")
+        sourceObj[val[0]] = val[1]
+    };
 
-    generateNWPolicyYaml(sourceid, targetid)
+    // variables
+    var port = "8080";
```

Comment:
we will have to take care of these ports that we are hardcoding.

Reply:
so that will be user input or we will be getting from workloads